charts/team-ns/templates/external-secrets.yaml

```diff
@@ -3,49 +3,80 @@
 {{- if get (index $v.apps "external-secrets") "enabled" }}
 {{- range $s := $v.secrets }}
 {{- $ns := index $s "namespace" | default $.Release.Namespace }}
-{{- $pathPrefix := printf "/secret/data/teams/team-%s" $v.teamId }}
+{{- $pathPrefix := printf "secret/data/teams/team-%s" $v.teamId }}
```

Comment:
Hashicorp Vault is strict about paths. Are you sure that the `/` is not needed ?

Reply:
They changed it and now it is without first slash. Luckily I found out rather quickly.